### PR TITLE
Bring environment in line with suggestion in #2

### PR DESCRIPTION
**Pull request by bame-da**
This is based on PR #2, but hand-crafted by me as we are not allowed to accept external contributions.

### CODE REVIEW

File: environments/daml-sdk-env/build/1_installprerequisites.sh.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# Following https://docs.azul.com/zulu/zuludocs/ZuluUserGuide/PrepareZuluPlatform/AttachAPTRepositoryUbuntuOrDebianSys.htm
+sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 0xB1998361219BD9C9
```
Comment (cocreature) [inline environments/daml-sdk-env/build/1_installprerequisites.sh.sh]: I would generally prefer `adoptopenjdk` over Zulu but I’ll leave it up to you. Just seems somewhat more official and it’s clearer how the distributions are built.

File: environments/daml-sdk-env/build/1_installprerequisites.sh.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+sudo apt-add-repository 'deb http://repos.azulsystems.com/ubuntu stable main'
+
+# Following https://classic.yarnpkg.com/en/docs/install/#debian-stable
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
```
Comment (cocreature) [inline environments/daml-sdk-env/build/1_installprerequisites.sh.sh:8]: Is that for the new GTG or is there something else where we need this?

File: environments/daml-sdk-env/build/3_configsdk.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+echo "PATH=\$PATH:\$HOME/.daml/bin" >> ~/.bashrc
+source ~/.bashrc
+
+# uninstall unnecessary VSCode extensions
+rm -rf /opt/.katacodacode/extensions/redhat.vscode-openshift-connector-0.0.19
+
+# install the daml extension (the hard way)
+cd /tmp & mkdir daml-bundled
+unzip $(find ~/.daml/sdk -d 1)/studio/daml-bundled.vsix -d daml-bundled
```
Comment (cocreature) [inline environments/daml-sdk-env/build/3_configsdk.sh:11]: I guess catacoda’s vscode server does not have something like `--install-extension`?

**Review by cocreature — APPROVED** [review]
Nice, thank you!

File: environments/daml-sdk-env/build/1_installprerequisites.sh.sh (hunk 1 above)
Comment (bame-da) [reply environments/daml-sdk-env/build/1_installprerequisites.sh.sh]: switched

File: environments/daml-sdk-env/build/1_installprerequisites.sh.sh (hunk 2 above)
Comment (bame-da) [reply environments/daml-sdk-env/build/1_installprerequisites.sh.sh:8]: Yes, this is for the GSG

File: environments/daml-sdk-env/build/3_configsdk.sh (hunk 3 above)
Comment (bame-da) [reply environments/daml-sdk-env/build/3_configsdk.sh:11]: It does, but it's somewhat custom so I'll start like this and improve once a first version is running.

**Pull request merged**